website/src/site/site.xml

```diff
@@ -51,7 +51,7 @@ under the License.
     <href></href>
   </bannerLeft>
   <bannerRight>
-    <src>apache-feather.gif</src>
+    <src>apache-feather.png</src>
```

Comment:
This is the original ones.
http://apache.org/img/asf_logo.png  (Full logo)
http://apache.org/img/feather_glyph_notm.png  (Feather logo)